app/models/topic.rb

```diff
@@ -0,0 +1,12 @@
+# == Schema Information
+#
+# Table name: topics
+#
+#  id         :bigint           not null, primary key
+#  name       :string           not null
+#  image      :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+class Topic < ApplicationRecord
```

Comment:
Maybe here we can add some validation, for example name presence.

See https://guides.rubyonrails.org/active_record_validations.html